dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -140,6 +141,24 @@ public void TabSizeAndScrollButtonsTest()
                 // With largerTab now rendering wider, the scroll buttons should appear:
                 Verify.IsTrue(AreScrollButtonsVisible(), "Scroll buttons should appear");
 
+                // Scroll all the way to the left and verify decrease/increase button visual state
+                FindElement.ByName<Button>("ScrollTabViewToTheLeft").InvokeAndWait();
+                Thread.Sleep(TimeSpan.FromMilliseconds(300));
+                Verify.IsFalse(IsScrollDecreaseButtonEnabled(), "Scroll decrease button should be disabled");
+                Verify.IsTrue(IsScrollIncreaseButtonEnabled(), "Scroll increase button should be enabled");
+
+                // Scroll to the middle position and verify decrease/increase button visual state
+                FindElement.ByName<Button>("ScrollTabViewToTheMiddle").InvokeAndWait();
+                Thread.Sleep(TimeSpan.FromMilliseconds(300));
+                Verify.IsTrue(IsScrollDecreaseButtonEnabled(), "Scroll decrease button should be enabled");
+                Verify.IsTrue(IsScrollIncreaseButtonEnabled(), "Scroll increase button should be enabled");
+
+                // Scroll all the way to the right and verify decrease/increase button visual state
+                FindElement.ByName<Button>("ScrollTabViewToTheRight").InvokeAndWait();
+                Thread.Sleep(TimeSpan.FromMilliseconds(300));
```

Comment:
I just noticed that you are using Thread.Sleep here. It might be better to use `Wait.ForMilliseconds`.

Reply:
Done.